client/src/hooks/useStepList/useMemberAction.tsx

```diff
@@ -0,0 +1,32 @@
+import type {BillStep, MemberAction, MemberStep} from 'types/serviceType';
+
+import useEventId from '@hooks/useEventId/useEventId';
+import {requestDeleteMemberAction} from '@apis/request/member';
+
+interface UseMemberActionProps {
+  stepList: Array<BillStep | MemberStep>;
+  refreshStepList: () => Promise<void>;
+}
+
+const useMemberAction = ({stepList, refreshStepList}: UseMemberActionProps) => {
+  const {eventId} = useEventId();
+  const memberActions = stepList.filter(step => step.type !== 'BILL').flatMap(step => step.actions);
+
+  const hasNextMemberAction = (name: string, sequence: number) => {
+    return memberActions.find(action => action.name === name && action.sequence > sequence) !== undefined;
```

Comment:
```suggestion
    return memberActions.some(action => action.name === name && action.sequence > sequence);
```

find보다 some 메소드를 사용하는 것은 어떨까요?

Reply:
아닛 그런 방법이~!~!~!~ 너무 좋은 의견이에요
코드가 바뀌긴했지만 이런 상황나오면 some 사용해야지!